Weather App/viewModel/WeatherForCityViewModel.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  WeatherForCityViewModel.swift
+//  Weather App
+//
+//  Created by Piotr Chlapinski on 20/07/2022.
+//
+
+import CoreLocation
+import Foundation
+import Keys
+
+class WeatherForCityViewModel {
+    var weathers: [WeatherForCity] = []
+
+    func getWeatherForCity(cityName: String) {
+        guard let url = URL(string: "https://api.openweathermap.org/data/2.5/weather?q=\(cityName)" +
+                            "&appid=\(EidolonKeys().openWeatherApiKey)&units=metric") else {
+            return
+        }
+        getWeather(url: url)
+    }
+
+    func getWeatherForCoordinate(coordinate: CLLocationCoordinate2D) {
+        guard let url = URL(string: "https://api.openweathermap.org/data/2.5/weather?lat=\(coordinate.latitude)" +
```

Comment:
Could you keep base of url `https://api.openweathermap.org/data/2.5/weather` in some configuration or const value, it is the same in each request